src/main/java/function/config/RedisConfig.java

```diff
@@ -0,0 +1,20 @@
+
+package function.config;
+
+import redis.clients.jedis.Jedis;
+
+/** Redis configuration. */
+public class RedisConfig {
+
+  private static Jedis client;
+  private static final String JWKS_HOST = System.getenv("REDISHOST");
+  private static final int JWKS_PORT = Integer.parseInt(System.getenv("REDISPORT"));
```

Comment:
Will `REDISHOST` & `REDISPORT` env variable part of gateway configuration?

Reply:
Yes We can configure those details there